src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -1,4 +1,7 @@
 import * as React from 'react';
+import getProp from 'lodash/get';
+import { matchPath } from 'react-router-dom';
+import { History } from 'history';
```

Comment:
I don't think we currently import from `history` anywhere else, so this could complicate the parent app's dependencies. Can we import `RouterHistory` from `react-router-dom`, instead?

Reply:
It doesn't appear that `react-router-dom` has any exported History types -- seems like that worked for flow types but not typescript?

Reply:
Gotcha. Can we bring in https://www.npmjs.com/package/@types/history as a dev dependency to avoid the explicit `history` dependency?